### PR TITLE
Get a node topology zone from a label and save to a file

### PR DESCRIPTION
**Pull request by Ricardo-Osorio**
Because C2 will require the node topology label when starting up and api-manager is too slow at doing this the init container now fetches and stores this information locally (text file)

### CODE REVIEW

**Comment by Arau** [comment]
Looking at this change, it just rang a bell. I have the impression that this change would require role permissions to be added to the service account. I might be wrong, but I believe that accessing the api from the daemonset pod has never been done before, and as a result the perms are not there.

**Comment by Ricardo-Osorio** [comment]
> require role permissions to be added to the service account

I did run this fine on a cluster but what I can do is test this again today on a fresh cluster because, at some point, I did add the permissions to list nodes and then removed it an it kept working. Perhaps because they were "cached" or similar (I was reusing the same cluster over and over again)

**Comment by Ricardo-Osorio** [comment]
Follow up: seems like the permissions are not needed and it runs fine as it is 👍

File: init.go [hunk 1]
```diff
@@ -88,6 +89,22 @@ func main() {
...
+
+	node, err := kubeclient.CoreV1().Nodes().Get(os.Getenv("NODE"), v1.GetOptions{})
+	if err != nil {
+		log.Fatalf("failed to get node from name \"%s\": %v", os.Getenv("NODE"), err)
+	}
+
+	file, err := os.Create("/var/lib/storageos/init_envs.sh")
+	if err != nil {
+		log.Fatalf("failed to create file init_envs.sh: %v", err)
+		os.Exit(1)
```
Comment (Mojachieee) [inline init.go]: nit: `log.Fatal` will already call `os.Exit`, so there's no need for the extra one here

**Review by Mojachieee — DISMISSED** [review]
LGTM - ~~though we may need to make permission changes (but they will need to be done in the operator)~~

Looks like the storageos-node service account already has the permissions necessary due to:
https://github.com/storageos/operator/blob/release/v2.8.2-beta.2/channels/packages/node/2.5.0/driver-registrar-cluster-role.yaml#L4

File: init.go (hunk 1 above)
Comment (Ricardo-Osorio) [reply init.go]: Done

**Review by Mojachieee — APPROVED** [review]

**Pull request merged**